eth/core/precompile/factories.go

```diff
@@ -125,6 +125,9 @@ func buildIdsToMethods(si StatefulImpl, contractImpl reflect.Value) (map[string]
 		if err != nil {
 			return nil, err
 		}
+		if methodName == "" {
```

Comment:
i think this should revert then? 

or do we want to allow it? 

(i guess good for helpers)

Reply:
Yea basically it just skips any function that’s not in the ABI, which we should’ve had from the beginning